### PR TITLE
Build v 0.21.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- build script updated to include --no-build-isolation and --no-deps.
 - Removed redundant dependencies from buil.
 - Pinned dependencies in host.
 - Enhanced 'about' section.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,25 +12,23 @@ source:
...
+    - scikit-learn 1.2.1
+    - setuptools_scm 7.0.4
+    - packaging 
     - python
+    - pip
+    - wheel
+    - setuptools
   run:
     - python
     - {{ pin_compatible('numpy') }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:34]: `    - importlib_resources >=1.3  # [py<39]` instead of    ` - importlib_resources >=1.3  # [py <= 38]`

**Review by skupr-anaconda — APPROVED** [review]
LGTM but `importlib_resources`

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml:34]: But this is the same thing.

**Review by sumit0190 — APPROVED** [review]

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**